### PR TITLE
XEP-0045 - Fix typo missed in rev. 1.25

### PR DESCRIPTION
**Pull request by licaon-kter**
Ping @stpeter

### CODE REVIEW

File: xep-0045.xml [hunk 1]
```diff
@@ -4076,7 +4076,7 @@
   </query>
 </iq>
 ]]></example>
-      <p>Note: The _whois configuration option specifies whether the room is non-anonymous (a value of "anyone"), semi-anonymous (a value of "moderators"), or fully anonmyous (a value of "none", not shown here).</p>
+      <p>Note: The _whois configuration option specifies whether the room is non-anonymous (a value of "anyone"), semi-anonymous (a value of "moderators"), or a value of "none" (not shown here).</p>
```
[the remark this answers is not in the document]
Comment (licaon-kter) [reply xep-0045.xml:4079]: Two places, in "anonymous" and in existence of the mention of "fully anonymous" which should have been removed in 1.25 according to the history entries.

[the remark this answers is not in the document]
Comment (horazont) [reply xep-0045.xml:4079]: But then the value of `"none"` should be removed from the sentence altogether, this was what was mainly confusing me.

[the remark this answers is not in the document]
Comment (licaon-kter) [reply xep-0045.xml:4079]: i can't comment on that :)

[the remark this answers is not in the document]
Comment (stpeter) [reply xep-0045.xml:4079]: Yes, I think we should remove "none" as well.

[the remark this answers is not in the document]
Comment (licaon-kter) [reply xep-0045.xml:4079]: Will do, thanks for the clarification

[the remark this answers is not in the document]
Comment (horazont) [reply xep-0045.xml:4079]: @licaon-kter While you’re at it, council briefly discussed this, and we’d prefer to have a note somewhere that `"none"` was formerly an allowed value. There may still be deployments in the wild with that value and implementors may need to be aware about that.

[the remark this answers is not in the document]
Comment (stpeter) [reply xep-0045.xml:4079]: Agreed - remove it as a live option but mention that it was formerly included.

[the remark this answers is not in the document]
Comment (licaon-kter) [reply xep-0045.xml:4079]: Great

**Comment by licaon-kter** [comment]
Any news?

**Comment by stpeter** [comment]
LGTM!

**Comment by licaon-kter** [comment]
Any issues here?

**Pull request merged**